BuildaGitServer/Base/Authentication.swift

```diff
@@ -69,4 +73,16 @@ extension ProjectAuthenticator: KeychainStringSerializable {
             self.secret
             ].joinWithSeparator(":")
     }
+
+    internal static func createEnterpriseService(host: String) -> GitService? {
+        guard let url = NSURL.init(string: "http://\(host)") else { return nil }
+        do {
+            let response = try NSString.init(contentsOfURL: url, encoding: NSASCIIStringEncoding)
+            if response.lowercaseString.containsString("github") {
```

Comment:
Can you walk me through what the response from Enterprise GitHub looks like? Does it return a string that says `github`?